### DISCUSSION TITLE
O mítico homem-mês: ensaios sobre engenharia de software

### DISCUSSION THREAD

**Discussion by gilsondev** [pilhacheia/clube-do-livro-dev#25] [Books]
Esse livro é um daqueles clássicos que sempre é recomendado. Olhem a sinopse:

> Poucos livros sobre gestão de projetos de software têm sido tão influentes e perenes quanto O Mítico Homem-Mês. Com uma rica mistura de fatos sobre engenharia de software e opiniões que provocam a reflexão, Frederick Brooks oferece sua visão para todos aqueles que lidam com a gestão de projetos complexos. Os ensaios deste livro vêm diretamente da experiência de Frederick como gerente de projetos para a família de computadores System/360 da IBM e, posteriormente, com o OS/360, seu massivo sistema de software. Passados 20 anos da publicação original, em 1975, Brooks revisou suas ideias originais, adicionando novos pensamentos e conselhos em uma edição especial de aniversário. Agora, quase 35 anos depois da primeira edição, comemorando sua primeira versão em português, Brooks concede uma entrevista exclusiva sobre seu trabalho, que os leitores podem conferir nas primeiras páginas deste livro.
> 
> O argumento central de O Mítico Homem-Mês é o de que grandes projetos de programação sofrem de problemas de gestão cuja natureza difere dos projetos pequenos em função da divisão das tarefas; a integridade conceitual de um produto é um fator crítico em seu desenvolvimento; e que é difícil, mas possível, atingir tal integridade. Seu ensaio seminal de 1986, "Não existe bala de prata" também está aqui, complementado por um novo texto da edição de 1995, onde Brooks afirma que "Não haverá nenhuma bala de prata no intervalo de dez anos". Estes dez anos já se passaram e Brooks continua atual.

[Link da Amazon](https://www.amazon.com.br/m%C3%ADtico-homem-m%C3%AAs-ensaios-engenharia-software/dp/8550802530): R$ 66,95

**Comment by lincolixavier** [comment]
Esse com certeza tem que ser lido!

**Reply by hsmiranda** [reply]
Tá na minha lista para ler.

**Comment by lincolixavier** [comment]
Boa pedida pra começar 2024!
